serde-helpers/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Small helpers and utilities for serde and JSONSchema.
+
+use schemars::JsonSchema;
+use serde::Deserialize;
+use serde::Serialize;
+
+#[derive(JsonSchema, Serialize, Deserialize)]
```

Comment:
if you want this to turn into a `Result` in the clients you could add the x-rust-type extensions

Reply:
Added in 52bd108a4